vtysh/bgp_vty.c

```diff
@@ -5082,11 +5099,32 @@ DEFUN(no_neighbor_strict_capability,
     return CMD_SUCCESS;
 }
 
+static void
+neighbor_set_timers(const struct ovsrec_bgp_neighbor *bgpn, timer_val_t *tim_values, int tim_number)
+{
+    const struct ovsrec_bgp_neighbor *bgpn_peer_group, *bgpn_next;
+    char *key_timers[2];
+    key_timers[0] = OVSDB_BGP_TIMER_KEEPALIVE;
+    key_timers[1] = OVSDB_BGP_TIMER_HOLDTIME;
+
+    ovsrec_bgp_neighbor_set_timers(bgpn, key_timers,
+                                           (int64_t *)tim_values, tim_number);
+
+    if (object_is_bgp_peer_group(bgpn)) {
+        bgpn_peer_group = bgpn;
+        OVSREC_BGP_NEIGHBOR_FOR_EACH_SAFE(bgpn, bgpn_next, idl) {
```

Comment:
propose use a macro OVSREC_BGP_NEIGHBOR_FOR_EACH(bgpn, idl) and to exclude variable bgpn_next.